### ISSUE TITLE
Bugsets seems to have broken post-Heroku

### ISSUE THREAD

**Issue by ehashman** [openhatch/oh-mainline#1762]
The bugsets page in production (e.g. [this test bugset I just created](https://openhatch.org/bugsets/4-a-post-heroku-test-event)) seems to have broken, possibly since the Heroku migration. The bugs won't expand, and inplaceedit doesn't seem to be working any more. I was going to submit an issue suggesting we link this somewhere so people can find it, but maybe not in its current state :\

**Comment by waseem18** [comment]
The problem is actually with 'inplaceedit' . I have started working on this issue. I'll try to resolve it as soon as I can.

**Comment by ehashman** [comment]
I think it's not just inplaceedit, as the bugs wouldn't expand when clicked on. To me, that indicates worse js problems. Let me know if you find anything.

**Comment by waseem18** [comment]
It works fine when I run OH locally. Every button is working the way it should be. So I was thinking if the issue was due to recent Heroku migration. What do you say?

**Comment by ehashman** [comment]
Yeah, there might be some problems with static stuff. I recall that being an issue when I first introduced the vendor packages.

**Comment by jwm** [comment]
The commit I just pushed should fix this; the automatic Heroku run of `manage.py collectstatic` wasn't picking up on the production settings file, so these static files were being put in the wrong place.

I noticed that the project icon (https://openhatch.org/static/images/icons/projects/b26edd33cb334f32b3f01681d3229109.png in this case) still isn't loading. It looks like something was writing project icons to `~deploy/milestone-a/mysite/static/images/icons/projects` on linode.openhatch.org, but I can't find what was doing that. The most recent timestamp on any of the icons is May 2015, so maybe something broke or was removed around that time? That location won't be used any more, so they'll need to be generated or served via Heroku somehow.
